app/src/main/java/com/sapuseven/untis/fragments/TimetableItemDetailsFragment.kt

```diff
@@ -61,6 +61,20 @@ class TimetableItemDetailsFragment(item: TimegridItem?, timetableDatabaseInterfa
 		} ?: throw IllegalStateException("Activity cannot be null")
 	}
 
+	override fun onStart() {
+		super.onStart()
+		if (activity is MainActivity) {
+			(activity as MainActivity).setFullscreenDialogActionBar()
+		}
+	}
+
+	override fun onStop() {
+		super.onStop()
+		if (activity is MainActivity) {
+			(activity as MainActivity).setDefaultActionBar()
+		}
+	}
```

Comment:
Minor thing:
Don't use `{ }` for small `if`-statements like this. It improves readability.